src/components/DocumentationTopic/Summary/Availability.vue

```diff
@@ -19,39 +18,39 @@
         class="platform"
         :change="!!changeFor(platform.name)"
       >
-        <AvailabilityRange
-          :deprecatedAt="platform.deprecatedAt"
-          :introducedAt="platform.introducedAt"
-          :platformName="platform.name"
-        />
-        <Badge v-if="platform.deprecatedAt" variant="deprecated" />
-        <Badge v-else-if="platform.beta" variant="beta" />
+        <AvailabilityBadge>
+          <AvailabilityRange
+            :deprecatedAt="platform.deprecatedAt"
+            :introducedAt="platform.introducedAt"
+            :platformName="platform.name"
+          />
+        </AvailabilityBadge>
+        <AvailabilityBadge v-if="platform.deprecatedAt" variant="deprecated" />
+        <AvailabilityBadge v-else-if="platform.beta" variant="beta" />
       </Item>
     </List>
   </Section>
 </template>
 
 <script>
-import Badge from 'docc-render/components/Badge.vue';
 import { getAPIChanges } from 'docc-render/mixins/apiChangesHelpers';
 import { ChangeTypes } from 'docc-render/constants/Changes';
 import AvailabilityRange from './AvailabilityRange.vue';
 import List from './List.vue';
 import ListItem from './ListItem.vue';
 import Section from './Section.vue';
-import Title from './Title.vue';
+import AvailabilityBadge from './AvailabilityBadge.vue';
```

Comment:
What is the need to create a new Badge component? It looks almost identical to me.

Reply:
Hanqing and I spoke about it and she will work on the `Badge` component instead of creating a new one.